netlify-functions-ecommerce/frontend/Cart/cart.js

```diff
@@ -0,0 +1,72 @@
+export default {
+  data: function() {
+    return {
+      products: [],
+      cart: [],
+    }
+  },
+  methods: {
+    // a simple solution to handle the weird rendering of the number on adding and removing is pass a temp copy to the backend instead of the original.
+    async removeAll(item) {
+      const updatedCart = await fetch('http://localhost:8888/.netlify/functions/removeFromCart', {
```

Comment:
Don't hard code `http://localhost:8888/.netlify/functions` on every request . Move that into a `constants.js` file.

Reply:
I'm not sure how to import because it causes an error when using `require`

Reply:
This is the error when using import:
`Loading module from “http://localhost:3000/.config/constants” was blocked because of a disallowed MIME type (“text/html”).`
This is the error when using require
`Uncaught ReferenceError: require is not defined`

Reply:
In that case, just move `'http://localhost:8888/.netlify/functions'` to a `const serverUrl = 'http://localhost:8888/.netlify/functions'` variable at the top of every file. So at least we only have 1 per file.